install/operator/deploy/manifests/1.7.x/syndesisoperator.1.7.x.clusterserviceversion.yaml

```
@@ -0,0 +1,933 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: syndesisoperator.1.7.x
+  namespace: placeholder
+  annotations:
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    certified: "false"
+    createdAt: 2019-05-08T16:12:00Z
+    containerImage: syndesis/syndesis-operator
+    support: Red Hat Fuse Online
```

```suggestion
    support: Syndesis
```

Not sure actually what value should this hold? Can't find any documentation in [the reference](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md) for the annotations used.